karspexet/settings.py

```diff
@@ -29,12 +44,12 @@
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '&-)aly8rq=l8-7193rj0e@p$tp571+q5&g0jyi8#)u!rt-!=b8'
+SECRET_KEY = ENV.get("SECRET_KEY", "&-)aly8rq=l8-7193rj0e@p$tp571+q5&g0jyi8#)u!rt-!=b8")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = ENV.get("SECRET_KEY", True)
```

Comment:
Jag är nästan lite av åsikten att jag skulle vilja vända på denna, och stoppa in en `"debug": true`-rad i en sample-env.json istället, for att försäkra oss att vi inte kör med fel inställning i produktion. Vad tror du om det?

Reply:
Min tanke är att det är en bra grundregel att göra default-värdena anpassade för Dev-miljön så långt det går. Fördelar med det är:
* En ny utvecklare behöver redigera / förstå mycket mindre av sin `env.json`
* En gammal utvecklare behöver inte redigera sin `env.json` lika ofta om det kommer till ny funktionalitet utan kan lita på att default-värdena gör att koden fortfarande går att köra.
* Prod-miljön kommer ändå behöva noggrann config med lösenord, andra hemligheter, och prod-specifik config som inte en "vanlig" utvecklare har så stort behov av att känna till.

Så att i prod behöva skriva in `"DEBUG": true` är alltså inte så stor skillnad från att ändå behöva skriva in `ALLOWED_HOSTS` eller `SECRET_KEY`, tänker jag. Så om vi skall optimera för att någon miljö skall kunna ha liten konfig har utvecklingsmiljön mycket som talar för sig.

Inser dock att jag skrev fel på nyckelvärdet här! Så klart ska det vara `DEBUG = ENV.get("DEBUG", True)`